docs/requirements.txt

```diff
@@ -1,6 +1,6 @@
 breathe==4.13.1
 docutils==0.14
-Pygments==2.7.4
+Pygments==2.3.1
```

Comment:
I think that this needs to be 2.7.4. Not sure how the merge did not deal with this automatically?